.pre-commit-config.yaml

```diff
@@ -6,9 +6,15 @@ repos:
         entry: poetry run black
         language: system
         types: [file, python]
-    
+
       - id: isort
         name: isort
         entry: poetry run isort
         language: system
         types: [file, python]
+
+      - id: yamllint
```

Comment:
Can you add docs on these in the contibuting file?

Reply:
I swapped out `yamllint` with `yamlfix` which lets us automatically format YAML files in the pre-commit hook, just like we do for python files. I also mention YAML formatting requirements and options in `CONTRIBUTING.md`. Let me know if it looks ok, or if we need to make other changes.